frontend/src/metabase/visualizations/visualizations/CartesianChart/use-chart-events.ts

```diff
@@ -86,6 +87,16 @@ export const useChartEvents = (
             return;
           }
 
+          if (
+            event.name === "goal-line" &&
+            event.event?.target?.style?.text === settings["graph.goal_label"]
```

Comment:
The line and label trigger hover similar hover events. Is there a way to make only the label cause hover events to be triggered besides manually checking the element that triggered the event?

For example, without this manual check all hovers over the line and label cause tooltips to be created.

**Without Manual Check**

https://github.com/metabase/metabase/assets/22608765/9bf3f1f6-e117-4774-a7af-2e56baff45fb

**With Manual Check**


https://github.com/metabase/metabase/assets/22608765/a00e9b65-eb45-40a1-80e9-c8d55b41bb9c